thorlcr/thorutil/thmem.cpp

```diff
@@ -345,7 +363,11 @@ class CSharedSpillableRowSet : public CSpillableStreamBase, implements IInterfac
                 rwFlags |= rw_grouped;
             return ::createRowStream(spillFile, rowIf, rwFlags);
         }
-        return new CStream(*this);
+        rowidx_t toRead = rows.numCommitted();
```

Comment:
Then it would not get it here.
spillFile would have been set and the proceeding block would be entered.